### PR TITLE
OLM Configuration Updates

### PR DESCRIPTION
**Pull request by stevekuznetsov**
periodic-auto-olm-downstreaming: report failures to slack:

Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

---

openshift/operator-framework-olm: publish the ci image

Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

---

openshift/operator-framework-olm: run periodics, add them to testgrid

Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

---

### CODE REVIEW

File: ci-operator/jobs/infra-periodics.yaml [hunk 1]
```diff
@@ -1200,6 +1200,12 @@ periodics:
     ci.openshift.io/role: infra
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-auto-olm-downstreaming
+  reporter_config:
+    slack:
+      channel: '#forum'
```
Comment (bparees) [inline ci-operator/jobs/infra-periodics.yaml:1205]: ??

File: ci-operator/config/openshift/operator-framework-olm/openshift-operator-framework-olm-master.yaml [hunk 2]
```diff
@@ -51,6 +51,7 @@ promotion:
   namespace: ocp
   to:
   - additional_images:
+      ci-image: ci-image
```
Comment (bparees) [inline ci-operator/config/openshift/operator-framework-olm/openshift-operator-framework-olm-master.yaml:54]: give this an olm specific name?

Comment (stevekuznetsov) [reply ci-operator/config/openshift/operator-framework-olm/openshift-operator-framework-olm-master.yaml:54]: it's promoted to an olm-specific namespace

**Pull request merged**